### PR TITLE
aggregator: make buffer size configurable.

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

Increasing the aggregator buffer size is a trade-off between RSS usage
and performance, let's make it configurable to be able to tweak that
using a configurable parameter.

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -142,6 +142,13 @@ api_key:
 ## 'aggregator_stop_timeout' to 0.
 #
 # aggregator_stop_timeout: 2
+#
+# @param aggregator_buffer_size - unsigned int
```
Comment (l0k0ms) [inline pkg/config/config_template.yaml]:
```suggestion
# @param aggregator_buffer_size - integer - optional - default: 100
```

Comment (remeh) [reply pkg/config/config_template.yaml]: Thanks!

**Comment by l0k0ms** [comment]
G2G doc wise

**Review by ogaca-dd — APPROVED** [review]
LGTM

**Pull request merged**